specs/darwin/gatekeeper.table

```diff
@@ -0,0 +1,9 @@
+table_name("gatekeeper")
+description("OS X Gatekeeper Details.")
+schema([
+    Column("assessments_enabled", INTEGER, "1 If a Gatekeeper is enabled else 0"),
+    Column("dev_id_enabled", INTEGER, "1 If a Gatekeeper allows execution from identified developers else 0"),
+    Column("version", TEXT, "Version of Gatekeeper's gke.bundle"),
+    Column("opaque_version", TEXT, "Version of Gatekeeper's gkopaque.bundle"),
+])
+implementation("gatekeeper@genGateKeeper")
```

Comment:
One last thing for both of these tables, as it seems your opening some sqlite DBs, would you mind trying to add in the `fuzz_paths` line to fuzz your tables? [crashes](https://github.com/facebook/osquery/blob/master/specs/darwin/crashes.table#L24-L27) has an example of how to do this, but I don't think we've thoroughly played with zzufs ability to fuzz sqlite DBs, though it looks like it can. Lemme know!